.github/workflows/run-recipe-test.yaml

```diff
@@ -0,0 +1,49 @@
+name: Run recipe test
+on:
+  repository_dispatch:
+    types: [run-recipe-test-command]
+jobs:
+  run-recipe-test:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the pull request branch
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.ACTIONS_BOT_TOKEN }}
```

Comment:
Can you explain what this token is for? It authorizes the action to do stuff it could not normally do?

Reply:
@rabernat The default `GITHUB_TOKEN` does not have permissions to create `repository_dispatch` [events](https://github.com/peter-evans/slash-command-dispatch#token).  In this case we are operating in the context of the `repository_dispatch` event (rather than the `pull_request` event so I believe this token is necessary to access the event [payload](https://github.com/pangeo-forge/staged-recipes/pull/28/files#diff-db5b2dc009168c4c406c6beffcff748f1649be4e83b736da9d7555e718910db6R15).  I'm not entirely sure this is the required token here but I was following the demonstration example provided in the slash command action [repo](https://github.com/peter-evans/slash-command-dispatch-processor/blob/master/.github/workflows/black-command.yml#L13-L14).